src/test/test.js

```diff
@@ -58,15 +58,18 @@ describe('watsonwork-echo', () => {
         return;
       }
     };
-
+  
     // Create the Echo Web app
-    echo.webapp('testappid', 'testsecret', 'testwsecret', (err, app) => {
-      expect(err).to.equal(null);
+    echo.webapp('testappid', 'testsecret', 'testwsecret').then((app) => {
+      expect(app).to.not.equal(null);
       check();
+    }).catch((err) => {
+      throw new Error('Could not create Web app');
     });
+
   });
 
-  it('handles Webhook challenge requests', (done) => {
+  it('handles Webhook challenge requests', async (done) => {
```

Comment:
I don't think using a done callback like this is actually compatible with an async test function.